.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+**Note: for support questions, please use Discord**. This repository's issues are reserved for feature requests and bug reports.
+
+* **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request => Please do not submit support request here, see note at the top of this template.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5470889)